reference/docs-conceptual/PowerShell-Core-Support.md

```diff
@@ -33,26 +33,46 @@ Eventually, we expect PowerShell Core will adopt the "long-term servicing" appro
 
 ## Supported platforms
 
-PowerShell Core is officially supported on the following platforms:
-
-* Windows 7, 8.1, and 10
-* Windows Server 2008 R2, 2012 R2, 2016
-* [Windows Server Semi-Annual Channel][semi-annual]
-* Ubuntu 14.04, 16.04, and 17.04
-* Debian 8.7+, and 9
-* CentOS 7
-* Red Hat Enterprise Linux 7
-* OpenSUSE 42.2
-* Fedora 27, 28
-* macOS 10.12+
+Please see the following table to see what platform the version of PowerShell Core you are using is officially supported:
+
+|                                                   | 6.0       | 6.1       |
+|---------------------------------------------------|-----------|-----------|
+| Windows 7, 8.1, and 10                            | Supported | Supported |
+| Windows Server 2008 R2, 2012 R2, 2016             | Supported | Supported |
+| [Windows Server Semi-Annual Channel][semi-annual] | Supported | Supported |
+| Ubuntu 14.04 and, 16.04                           | Supported | Supported |
+| Ubuntu 17.10, and 18.04                           |           | Supported |
+| Debian 8.7+, and 9                                | Supported | Supported |
+| CentOS 7                                          | Supported | Supported |
+| Red Hat Enterprise Linux 7                        | Supported | Supported |
+| OpenSUSE 42.2                                     | Supported | Supported |
+| Fedora 27                                         | Supported | Supported |
+| Fedora 28                                         |           | Supported |
+| macOS 10.12+                                      | Supported | Supported |
 
 Our community has also contributed packages for the following platforms,
-but they are not officially suppported:
+but they are not officially supported:
 
 * Arch Linux
```

Comment:
Would it be easier to read if you put these in the table and have it say `Community`?
Should we include `Experimental` for Raspbian on 6.0 and `Supported` for 6.1?
We'll have to update this table again once Alpine docker image is out or just do it now.

Reply:
Raspbian isn't supported by DotNet, so I say community:  https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md#linux

I think we should update Alpine when the work is done.

I'll update the table as suggested.